types/yup/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for yup 0.26
+// Type definitions for yup 0.27
```

Comment:
We generally don't wanna change the minor version of the package until Yup itself releases a minor version bump. Since they're still at 0.26, I think we should keep it at 0.26. The publishing mechanism automatically ups the patch version without us having to manage it.